docs/release-process.md

```diff
@@ -11,109 +11,89 @@
-   _merge time_, so, when sorting by some proxy criterion, some care should
-   be used to understand which PR's were _merged_ after the last release. 
-
-1. Once the blocker PRs have merged, the release manager will create a new tag
-   by manually triggering the [`CI` Github Actions workflow](https://github.com/stacks-network/stacks-core/actions/workflows/ci.yml)
-   against the `release/X.Y.Z.A.n` branch.
-   
-1. Once the release candidate has been built, and docker images, etc. are available,
-   the release manager will notify various ecosystem participants to test the release
-   candidate on various staging infrastructure:
-
-   1. Stacks Foundation staging environments.
-   1. Hiro PBC testnet network.
-   1. Hiro PBC mainnet mock miner.
-
-   The release candidate should be announced in the `#stacks-core-devs` channel in the
-   Stacks Discord. For coordinating rollouts on specific infrastructure, the release
-   manager should contact the above participants directly either through e-mail or
-   Discord DM. The release manager should also confirm that the built release on the
-   [Github releases](https://github.com/stacks-network/stacks-core/releases/)
-   page is marked as `Pre-Release`.
-
-1. The release manager will test that the release candidate successfully syncs with
-   the current chain from genesis both in testnet and mainnet. This requires starting
-   the release candidate with an empty chainstate and confirming that it synchronizes
-   with the current chain tip.
-
-1. If bugs or issues emerge from the rollout on staging infrastructure, the release
-   will be delayed until those regressions are resolved. As regressions are resolved,
-   additional release candidates should be tagged. The release manager is responsible
-   for updating the `develop -> master` PR with information about the discovered issues,
-   even if other community members and developers may be addressing the discovered
-   issues.
-
-1. Once the final release candidate has rolled out successfully without issue on staging
-   infrastructure, the tagged release shall no longer marked as Pre-Release on the [Github releases](https://github.com/stacks-network/stacks-core/releases/) page. 
-   Announcements will then be shared in the `#stacks-core-devs` channel in the
-   Stacks Discord, as well as the [mailing list](https://groups.google.com/a/stacks.org/g/announce).
-
-1. Finally, the release branch `release/X.Y.Z.A.n` will be PR'ed into the `master` branch, and once merged, a PR for `master->develop` will be opened. 
+The release must be timed so that it does not interfere with a _prepare phase_.
+The timing of the next Stacking cycle can be found [here](https://stx.eco/dao/tools?tool=2).
+A release should happen at least 24 hours before the start of a new cycle, to avoid interfering with the prepare phase.
```

Comment:
nit: merge this sentence with beginning of paragraph?

Reply:
updated